### PR TITLE
Allow LSNs

### PR DESCRIPTION
**Pull request by albertkol**
## Done

- Change NoticeSchema to allow LSNs too.

### CODE REVIEW

File: webapp/security/schemas.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ class NoticePackage(Schema):
 
 
 class NoticeSchema(Schema):
-    id = String(required=True, validate=Regexp(r"USN-\d{1,5}-\d{1,2}"))
+    id = String(required=True, validate=Regexp(r"(LSN-|USN-)\d{1,5}-\d{1,2}"))
```
Comment (nottrobin) [inline webapp/security/schemas.py:87]: why not

```suggestion
    id = String(required=True, validate=Regexp(r"(LSN|USN)-\d{1,5}-\d{1,2}"))
```

?

I'm gonna approve this anyway, this is just a suggestion.

**Review by nottrobin — APPROVED** [review]
Great work thanks!

File: webapp/security/schemas.py (hunk 1 above)
Comment (albertkol) [reply webapp/security/schemas.py:87]: @nottrobin no good reason.. :D  I used as an example the regex we used for cves where the - is included. :P

Comment (albertkol) [reply webapp/security/schemas.py]: Done.

**Pull request merged**